PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-#include <shlobj.h>
+#include <shlwapi.h>

```

Comment:
How about this change? Well, with `shlobj.h` I had a weird error that could be a result of includes order, redefinition or whatever. 
`findreplacedlg.cpp(1833): error C2664: 'BOOL PathRemoveFileSpecW(LPWSTR)': cannot convert argument 1 from 'generic_string' to 'LPWSTR'`
Including `shlwapi.h` instead fixes the problem.

Reply:
No idea about this error.
But I think it's better use shlwapi instead of shlobj.